frontend/src/pages/EmployeeProfile/EmployeeSkillsAndCourses/AddCourseButton/AddCourseDialog/CoursesList/CoursesList.tsx

```diff
@@ -1,20 +1,19 @@
 import { FC, BaseSyntheticEvent } from 'react';
 
-import { ICourse, CoursesListType, TCheckedCourse } from 'types/course';
+import { LOADER } from 'constants/loaderTypes';
+import { IShortCourseInfo, CoursesListType } from 'types/course';
 import Loader from 'components/Loader';
-import { Loaders } from 'enums/loader';
 
 import { CourseItem } from './CourseItem';
 import { StyledList, StyledHorizontalLine } from './styled';
 
```

Comment:
and interface name here

Reply:
I dont understand what you meen

Reply:
wrong displayed here, i leave a comment on a IProps interface name, but I got you decision. lets discuss about interface naming on a next meeting